code/lib/docs-tools/src/argTypes/docgen/extractDocgenProps.ts

```diff
@@ -86,8 +86,8 @@ function extractProp(
   docgenInfo: DocgenInfo,
   typeSystem: TypeSystem,
   createPropDef: PropDefFactory
-): ExtractedProp {
-  const jsDocParsingResult = parseJsDoc(docgenInfo.description);
+): ExtractedProp | null {
+  const jsDocParsingResult = parseJsDoc(docgenInfo.description, {});
   const isIgnored = jsDocParsingResult.includesJsDoc && jsDocParsingResult.ignore;
```

Comment:
It seems to me that we don't need to fill in the second parameter, because that overrides the default value.

Reply:
done 💹